activitypub.php

```diff
@@ -84,10 +84,13 @@ function plugin_init() {
 		\add_action( 'init', array( __NAMESPACE__ . '\Blocks', 'init' ) );
 	}
 
-	$debug_file = __DIR__ . '/includes/debug.php';
-	if ( \WP_DEBUG && file_exists( $debug_file ) && is_readable( $debug_file ) ) {
-		require_once $debug_file;
-		Debug::init();
+	// Load development tools.
+	if ( 'development' === wp_get_environment_type() ) {
+		$debug_file = __DIR__ . '/development/debug.php';
+		if ( \WP_DEBUG && file_exists( $debug_file ) && is_readable( $debug_file ) ) {
+			require_once $debug_file;
+			Debug::init();
```

Comment:
Do we need the `\WP_DEBUG && file_exists( $debug_file ) && is_readable( $debug_file )` line in dev? Or can we just assume it to be true?

This would remove debug from other environments where `WP_DEBUG` is true. Not sure if that's what we want?

Reply:
We can assume WP_DEBUG as true...  maybe we should also add a load.php file, to handle all files in there!

Reply:
🤔

can you verify that the env is development when you restart?

Reply:
> should we move it to a dotfolder?

I think it's fine to keep in a regular folder, like `bin`, `docs`, and others that don't get distributed.